### ISSUE TITLE
Incremental Analysis Warnings

### ISSUE THREAD

**Issue by wborn** [openhab/static-code-analysis#330]
When I use SAT 0.6.0-SNAPSHOT the build contains a lot of warnings about using "Incremental Analysis" to make analysis faster. Can we solve the warnings by using incremental analysis which also makes the analysis faster?

Example:

```
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
[WARNING] This analysis could be faster, please consider using Incremental Analysis: https://pmd.github.io/pmd-6.2.0/pmd_userdocs_getting_started.html#incremental-analysis
```

It is really spamming the output because it shows many of these lines again and again after each other.
